github/apps.go

```diff
@@ -0,0 +1,100 @@
+package github
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"golang.org/x/oauth2/jws"
```

Comment:
Turns out this package is "deprecated" and not intended for public use anymore. I am happy to add any third-party replacement (there are many of them). Do you have any particular preferences?

Reply:
Added a commit to this PR that replaces `golang.org/x/oauth2/jws` with the suggested `gopkg.in/square/go-jose.v2`.